### PR TITLE
A/V info update change

### PR DESCRIPTION
**Pull request by klaashoekstra94**
First of all, I've done some cleaning up of the code and tried to make things more consistent.
I also changed the way that the A/V info is updated. In the current code, the delay before querying the A/V information is reset on every display update. However, that means that if the display updates more frequently than the set delay (now 10 seconds), the A/V info will never be queried.
I changed this by only starting a delayed query when the previous query has been completed.

### CODE REVIEW

**Comment by OnFreund** [comment]
@klaashoekstra94 looks like your branch and @winterscar's are too far apart, and there are too many changes here. Can you perhaps both rebase on top of current dev so they're aligned?

**Comment by klaashoekstra94** [comment]
It looks like both are up to date with the dev branch again.
However, it seems like all changes in the onkyo component are removed from [@winterscar](https://github.com/winterscar/core/tree/async-onkyo) his branch now?

**Comment by klaashoekstra94** [comment]
I don't understand why, but somehow the changed files are not as I expected?
As, it says that `CODEOWNERS`, `manifest.json` and `requirements_all.txt` are not updated in `winterscar:async-onkyo`, but they should be?

**Comment by OnFreund** [comment]
@klaashoekstra94 I just fixed all the merge conflicts and @winterscar's branch is up to date. Please pull it and rebase your work on top of it.

**Comment by klaashoekstra94** [comment]
@OnFreund Aah thanks, all seems good now!

File: homeassistant/components/onkyo/media_player.py [hunk 1]
```diff
@@ -309,19 +309,15 @@ def process_update(self, update):
                 self._attributes.pop(ATTR_VIDEO_INFORMATION, None)
                 self._attributes.pop(ATTR_PRESET, None)
                 self._attributes.pop(ATTR_VIDEO_OUT, None)
+
```
Comment (OnFreund) [inline homeassistant/components/onkyo/media_player.py:312]: I don't think we need the extra space here

File: homeassistant/components/onkyo/media_player.py [hunk 2]
```diff
@@ -473,10 +458,41 @@ async def async_select_output(self, output):
         """Set hdmi-out."""
         self._update_avr("hdmi-output-selector", output)
 
+    def _update_avr(self, propname, value):
```
Comment (OnFreund) [inline homeassistant/components/onkyo/media_player.py]: Any reason to move `_update_avr` and `_query_avr`. I think it just complicates the diff

File: homeassistant/components/onkyo/media_player.py [hunk 3]
```diff
@@ -506,16 +522,12 @@ def _parse_video_inforamtion(self, video_information_raw):
         else:
             self._attributes.pop(ATTR_VIDEO_INFORMATION, None)
 
+    def _query_delayed_av_info(self):
+        if not self._query_timer:
+            self._query_timer = self.hass.loop.call_later(10, self._query_av_info)
```
Comment (OnFreund) [inline homeassistant/components/onkyo/media_player.py:519]: This does create a potential bug - if you change stuff quickly enough, the second change won't trigger a query, and the first one might be too quick and pick up on the old values or temp ones. I think we can live with it, though.

File: homeassistant/components/onkyo/media_player.py (hunk 1 above)
Comment (klaashoekstra94) [reply homeassistant/components/onkyo/media_player.py:312]: As far as I know this is the black formatting we must comply to?
(It checks the formatting when I do a commit)

Comment (OnFreund) [reply homeassistant/components/onkyo/media_player.py:312]: Got it

File: homeassistant/components/onkyo/media_player.py (hunk 3 above)
Comment (klaashoekstra94) [reply homeassistant/components/onkyo/media_player.py:519]: Well the problem I had with the timer being reset every time was that when the screen changed too often, the _query_av_info was never triggered. That's why I now only query a new one when the timer has finished. But if you know a better solution I'd like to hear it!

File: homeassistant/components/onkyo/media_player.py (hunk 2 above)
Comment (klaashoekstra94) [reply homeassistant/components/onkyo/media_player.py]: Hmm, not really a reason I guess, good point :-P

File: homeassistant/components/onkyo/media_player.py (hunk 3 above)
Comment (OnFreund) [reply homeassistant/components/onkyo/media_player.py:519]: Yeah, I think your approach makes sense, and is probably better than mine. Don't have a good way to resolve this. It's a shame the AVR doesn't trigger the updates correctly by itself.

**Pull request merged**